devices/IAnalogSensorToIWear/src/IAnalogSensorToIWear.cpp

```diff
@@ -519,7 +534,7 @@ bool IAnalogSensorToIWear::attach(yarp::dev::PolyDriver* poly)
 
 bool IAnalogSensorToIWear::detach()
 {
-    handler.interface = nullptr;
+    pImpl->handler = nullptr;
```

Comment:
Mmh what about all the existing sensors? Should we delete them here?

Reply:
After adding the reset of the sensor I was getting termination fault when stopping the device.
I noticed that when terminating, the following error was printed
```
The stored ISensor has not been yet allocated
```
(coming from https://github.com/lrapetti/wearables/blob/fix-IAnalogSensorToIWear/devices/IAnalogSensorToIWear/src/IAnalogSensorToIWear.cpp#L588)
moving the `reset` in the close function fixed the problem (https://github.com/robotology/wearables/pull/52/commits/dbc994f5bede22c72c18d14d8e7b98d9c86811b4).